jabgui/src/main/java/org/jabref/gui/frame/MainMenu.java

```diff
@@ -181,17 +182,18 @@ private void createMenu() {
 
                 new SeparatorMenuItem(),
 
-                // region: Sharing of the library
-
                 // TODO: Should be only enabled if not yet shared.
                 factory.createSubMenu(StandardActions.GIT,
+                        factory.createMenuItem(StandardActions.GIT_COMMIT, new GitCommitAction(dialogService, stateManager)),
+                        new SeparatorMenuItem(),
                         factory.createMenuItem(StandardActions.GIT_SHARE, new GitShareToGitHubAction(dialogService, stateManager))
                 ),
+                new SeparatorMenuItem(),
```

Comment:
Nitpick: as per the existing style, the separator item comes with a line separation
```suggestion
                
                new SeparatorMenuItem(),
```